client/packages/common/src/intl/locales/en/common.json

```diff
@@ -183,7 +183,7 @@
   "description.rnr-amc": "The Average Monthly Consumption is calculated using the Adjusted consumption value, using this R&R form and the previous two, if available",
   "description.rnr-approved-quantity": "The quantity approved to be ordered. Populated after the R&R form is finalised and authorized.",
   "description.rnr-consumed": "The quantity of this item distributed during this period (via Outbound Shipments or Prescriptions)",
-  "description.rnr-consumed-adjusted": "Shows the consumption adjusted to account for stock out days. =(Consumed x 30)/(30 - Stock out duration)",
+  "description.rnr-consumed-adjusted": "Shows the consumption adjusted to account for stock out days. =(Consumed x (365.25/12))/((365.25/12) - Stock out duration)",
```

Comment:
I wasn't sure what this 30 represented, assuming it's number of days in month but please correct me if I'm wrong

Reply:
This should be:

Shows the consumption adjusted to account for stock out days. = (Consumed x (Days in period  / (Days in period - Stock out duration))

Reply:
Should I just change it to this then? or do you want me to add TODO like the approved message?